ToboggonApp/Toboggon/Models/PurchaseHistory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Toboggan.Models
+{
+    public class PurchaseHistory
+    {
+        public int OrderTableId { get; set; }
+        public int ShopId { get; set; } //ShopId
```

Comment:
Do you want to leave the comments here for others to read? Or was this to help you build out your other functions?